From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 3 Sep 2010 16:42:01 -0400
Subject: [net] bnx2x: use DEFINE_PCI_DEVICE_TABLE()
Message-id: <20100903164201.25415.52260.stgit@brian.englab.brq.redhat.com>
Patchwork-id: 28086
O-Subject: [RHEL5.6 BZ572012 PATCH 11/46] bnx2x: use DEFINE_PCI_DEVICE_TABLE()
Bugzilla: 572012
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>

from upstream a3aa18842a5303fc28fcc4d57dbd16618bd830a0

diff --git a/drivers/net/bnx2x_main.c b/drivers/net/bnx2x_main.c
index d837ec7..2dc22e6 100644
--- a/drivers/net/bnx2x_main.c
+++ b/drivers/net/bnx2x_main.c
@@ -133,7 +133,7 @@ static struct {
 };
 
 
-static const struct pci_device_id bnx2x_pci_tbl[] = {
+static DEFINE_PCI_DEVICE_TABLE(bnx2x_pci_tbl) = {
 	{ PCI_VDEVICE(BROADCOM, PCI_DEVICE_ID_NX2_57710), BCM57710 },
 	{ PCI_VDEVICE(BROADCOM, PCI_DEVICE_ID_NX2_57711), BCM57711 },
 	{ PCI_VDEVICE(BROADCOM, PCI_DEVICE_ID_NX2_57711E), BCM57711E },
